django-stubs/db/backends/base/base.pyi

```diff
@@ -60,9 +65,9 @@ class BaseDatabaseWrapper:
     operators: MutableMapping[str, str]
     def __init__(self, settings_dict: dict[str, Any], alias: str = ...) -> None: ...
     def ensure_timezone(self) -> bool: ...
-    @property
+    @cached_property
```

Comment:
Let's not touch these at the moment, there are several PRs to handle property/cached_property.